_posts/2019-05-09-node-version-menager.md

```diff
@@ -4,7 +4,7 @@ title:     "Korzyści wynikające z użycia Node Version Manager"
 date:      2019-05-09 17:00:00 +0100
 published: true
 author:    kczechowski
-image:     assets/img/posts/2019-05-09-node-version-menager/Node-Version-Manager.jpg
+image:     /assets/img/posts/2019-05-09-node-version-menager/Node-Version-Manager.jpg
```

Comment:
u, a to ciekawe?

Reply:
Musiałem przeoczyć, ten slash powinien być wszędzie, tu go nie było. Przez to się zdjęcie w poście Krzyśka nie wyświetlało

Reply:
https://blog.consdata.tech/2019/05/09/node-version-menager.html